vim25/soap/client.go

```diff
@@ -686,10 +686,7 @@ func (c *Client) Download(u *url.URL, param *Download) (io.ReadCloser, int64, er
 	}
 
 	r := res.Body
-
-	if strings.HasSuffix(u.Path, "/") {
```

Comment:
This was intentional, to avoid the slim but possible case of hitting the gzip header sequence when downloading a regular file.  Is there an issue with having the client use a trailing "/" to specify a directory?  

Is this what you meant by "missing backslashes"?